scripts/src/lib/Template.svelte

```diff
@@ -131,6 +133,17 @@
             </p>
         </div>
 
+        <div>
+            <div class="option-container">
+                <input id="kotlinGradle" type="checkbox" class="option-input" bind:checked={useKotlinGradle} />
+                <label for="kotlinGradle" class="option-label">Gradle Kotlin DSL</label>
+            </div>
+            <p class="option-body">
+                Use <a href="https://docs.gradle.org/current/userguide/kotlin_dsl.html">Gradle Kotlin DSL</a> instead of Groovy.
+                This will generate <code>build.gradle.kts</code> and <code>settings.gradle.kts</code> files.
```

Comment:
I think a better explaination here might be handy, we need to keep in mind a lot of people using this may not know what Kotlin, Groovy or "DSL" is. We also need to be mindful about the many disadvangages to using the Kotlin DSL.

Maybe:


Use the [Kotlin DSL](https://docs.gradle.org/current/userguide/kotlin_dsl.html) for the Gradle build script. This option is only reccomended for advanced users familar with Kotlin and Gradle. This may result in slower build peformance and compability issues with some IDEs.

Reply:
I really don't know what should I write there, so I'll use your suggestion.